en/dev_setup/building_px4.md

```diff
@@ -137,6 +137,73 @@ Build commands for other boards are given the [board-specific flight controller
 [VSCode](../dev_setup/vscode.md) is the officially supported (and recommended) IDE for PX4 development.
 It is easy to set up and can be used to compile PX4 for both simulation and hardware environments.
 
+
```

Comment:
Not directly related, but might be worth mentioning `make list_config_targets` already above in `## Other Boards`, so you know what's available.

Reply:
Excellent idea. Done. 

I will leave the make target thingy in place for now. I might move it into concepts next week - right now I think this is "OK".